py3status/modules/bluetooth.py

```diff
@@ -55,47 +52,28 @@ class Py3status:
     # available configuration parameters
     cache_timeout = 10
     device_separator = '|'
-    format = '{name}'
-    format_no_conn = 'OFF'
-    format_no_conn_prefix = 'BT: '
-    format_prefix = 'BT: '
+    format = 'BT[: {format_device}]'
```

Comment:
Switching from `hcitool` to `dbus` changes the behavior.

With `hcitool`, we just run the command and work with the data regardless.
With dbus, we can't obtain the data so we get exception.

I mentioned this because we're changing to `dbus` so why not fix the format at same time?

Basically, if people use Bluetooth all times, they likely have it on autostart and running all times... Otherwise, few people who manually starts bluetooth only sometimes (for audio or whatever reason), they will likely notice this red `bluetooth` instead of former `OFF` regardless.